docs/sidebars.ts

```diff
@@ -2,82 +2,112 @@ import type { SidebarsConfig } from "@docusaurus/plugin-content-docs";
 
 const sidebars: SidebarsConfig = {
   apiSidebar: [require("./modules/python/sidebar.json")],
-  docsSidebar: [{
-    type: 'doc',
-    label: 'Getting Started',
-    id: 'usage/getting-started',
-  }, {
-    type: 'doc',
-    label: 'Troubleshooting',
-    id: 'usage/troubleshooting/troubleshooting',
-  }, {
-    type: 'doc',
-    label: 'Feedback',
-    id: 'usage/feedback',
-  }, {
-    type: 'category',
-    label: 'How-to Guides',
-    items: [{
+  docsSidebar: [
+    {
       type: 'doc',
-      id: 'usage/how-to/cli-mode',
-    }, {
+      label: 'Getting Started',
+      id: 'usage/getting-started',
+    },
+    {
       type: 'doc',
-      id: 'usage/how-to/headless-mode',
-    }, {
+      label: 'Troubleshooting',
+      id: 'usage/troubleshooting/troubleshooting',
+    },
+    {
       type: 'doc',
-      id: 'usage/how-to/custom-sandbox-guide',
-    }, {
+      label: 'Feedback',
+      id: 'usage/feedback',
+    },
+    {
+      type: 'category',
+      label: 'How-to Guides',
+      items: [
+        {
+          type: 'doc',
+          id: 'usage/how-to/cli-mode',
+        },
+        {
+          type: 'doc',
+          id: 'usage/how-to/headless-mode',
+        },
+        {
+          type: 'doc',
+          id: 'usage/how-to/custom-sandbox-guide',
+        },
+        {
+          type: 'doc',
+          id: 'usage/how-to/evaluation-harness',
+        },
+        {
+          type: 'doc',
+          id: 'usage/how-to/openshift-example',
+        }
+      ]
+    },
+    {
+      type: 'category',
+      label: 'LLMs',
+      items: [
+        {
+          type: 'doc',
+          label: 'Overview',
+          id: 'usage/llms/llms',
+        },
+        {
+          type: 'category',
+          label: 'Providers',
+          items: [
```

Comment:
The only change is adding this category and putting the docs under it. The rest were formatting changes regarding indentation.